### ISSUE TITLE
[Keyboard Navigation - WIX] Link control under the 'License Terms' section is not keyboard accessible

### ISSUE THREAD

**Issue by tgugler** [wixtoolset/issues#6233]
If this issue is a bug:

* Which version of WiX are you building with?

> 3.14.0.1119

* Which version of Visual Studio are you building with (if any)?

> Visual Studio 2017

* Which version of the WiX Toolset Visual Studio Extension are you building with (if any)?

> 1.0.0.4

* Which version of .NET are you building with?

> Not Applicable

* If the problem occurs when installing your packages built with WiX, what is the version of Windows the package is running on?

> Not Applicable

* Describe the problem and the steps to reproduce it.

User Experience: 
If the controls are not keyboard accessible, the AT user will face issue in accessing that control. The user will not able to execute the desired action and will face difficulty while navigating through the screen.

Repro Steps:
1. Launch the "NPS Extension For Azure MFA" setup file.
2. Navigate all the controls of "MICROSOFT SOFTWARE LICENSE TERMS" popup window.
3. Verify the issue.

Actual Result:
Link control under the 'License Terms' section is not keyboard accessible.

Expected Result:
Link control under the 'License Terms' section should be keyboard accessible.

[MAS2.1.1_Links are not keyboard accessible (1).wmv.zip](https://github.com/wixtoolset/issues/files/5043496/MAS2.1.1_Links.are.not.keyboard.accessible.1.wmv.zip)

**Comment by barnson** [comment]
https://github.com/wixtoolset/issues/issues/6040
